src/Analyzers/CSharp/Tests/UseInterpolatedString/CSharpUseInterpolatedStringTests.cs

```diff
@@ -0,0 +1,245 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.UseInterpolatedString;
+using Microsoft.CodeAnalysis.Editor.UnitTests.CodeActions;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.UseInterpolatedString;
+
+using VerifyCS = CSharpCodeFixVerifier<
+    CSharpUseInterpolatedStringDiagnosticAnalyzer,
+    CSharpUseInterpolatedStringCodeFixProvider>;
+
+public sealed class CSharpUseInterpolatedStringTests
+{
+    [Fact]
+    public Task KeepInterpolatedStrings() => new VerifyCS.Test
```

Comment:
Suggestion №1: You can use `VerifyCS.VerifyCodeFixAsync(codem fixedCode)` when you just need to specify input and output without additional configurations.
Suggestion №2: In roslyn codebase tests like this usually have block bodies and actually `await` the result of the test rather than return the task from it. Please visit other tests, see how things are done there and be consistent with the rest of the codebase, thanks!